src/__swaps__/screens/Swap/components/TokenList/TokenToBuyList.tsx

```diff
@@ -1,34 +1,221 @@
+import { Box, Inline, Stack, Text, TextIcon, useColorMode } from '@/design-system';
+import { palettes } from '@/design-system/color/palettes';
+import { userAssetsStore } from '@/state/assets/userAssets';
+import { COIN_ROW_WITH_PADDING_HEIGHT, CoinRow } from '@/__swaps__/screens/Swap/components/CoinRow';
 import { ListEmpty } from '@/__swaps__/screens/Swap/components/TokenList/ListEmpty';
+import { AssetToBuySectionId, useSearchCurrencyLists } from '@/__swaps__/screens/Swap/hooks/useSearchCurrencyLists';
+import { useSwapContext } from '@/__swaps__/screens/Swap/providers/swap-provider';
+import { ChainId } from '@/__swaps__/types/chains';
+import { SearchAsset } from '@/__swaps__/types/search';
+import { SwapAssetType } from '@/__swaps__/types/swap';
+import { parseSearchAsset } from '@/__swaps__/utils/assets';
+import { getStandardizedUniqueIdWorklet } from '@/__swaps__/utils/swaps';
+import { DEVICE_WIDTH } from '@/utils/deviceUtils';
+import { EXPANDED_INPUT_HEIGHT, FOCUSED_INPUT_HEIGHT } from '../../constants';
 import { ChainSelection } from './ChainSelection';
-import { useSearchCurrencyLists } from '../../hooks/useSearchCurrencyLists';
+
+const BUY_LIST_HEADER_HEIGHT = 20 + 10 + 8; // paddingTop + height + paddingBottom
+
+interface SectionHeaderProp {
+  color: string | undefined;
+  symbol: string;
+  title: string;
+}
+
+const SECTION_HEADER_INFO: { [id in AssetToBuySectionId]: SectionHeaderProp } = {
+  favorites: {
+    title: i18n.t(i18n.l.token_search.section_header.favorites),
+    symbol: '􀋃',
+    color: 'rgba(255, 218, 36, 1)',
+  },
+  bridge: {
+    title: i18n.t(i18n.l.token_search.section_header.bridge),
+    symbol: '􀊝',
+    color: undefined,
+  },
+  verified: {
+    title: i18n.t(i18n.l.token_search.section_header.verified),
+    symbol: '􀇻',
+    color: 'rgba(38, 143, 255, 1)',
+  },
+  unverified: {
+    title: i18n.t(i18n.l.token_search.section_header.unverified),
+    symbol: '􀇿',
+    color: 'rgba(255, 218, 36, 1)',
+  },
+  other_networks: {
+    title: i18n.t(i18n.l.token_search.section_header.on_other_networks),
+    symbol: '􀊫',
+    color: palettes.dark.foregroundColors.blue,
+  },
+};
+
+const getColorWorklet = (isDarkMode: boolean) => {
+  'worklet';
+  return palettes[isDarkMode ? 'dark' : 'light'].foregroundColors;
+};
+
+const getChainColorWorklet = (isDarkMode: boolean) => {
+  'worklet';
+  return {
+    [ChainId.mainnet]: getColorWorklet(isDarkMode).mainnet,
+    [ChainId.arbitrum]: getColorWorklet(isDarkMode).arbitrum,
+    [ChainId.optimism]: getColorWorklet(isDarkMode).optimism,
+    [ChainId.polygon]: getColorWorklet(isDarkMode).polygon,
+    [ChainId.base]: getColorWorklet(isDarkMode).base,
+    [ChainId.zora]: getColorWorklet(isDarkMode).zora,
+    [ChainId.bsc]: getColorWorklet(isDarkMode).bsc,
+    [ChainId.avalanche]: getColorWorklet(isDarkMode).avalanche,
+    [ChainId.blast]: getColorWorklet(isDarkMode).blast,
+    [ChainId.degen]: getColorWorklet(isDarkMode).degen,
+  };
+};
```

Comment:
move these to a utils file. might make more sense to move the `SECTION_HEADER_INFO` as well